var/spack/repos/builtin/packages/e2fsprogs/package.py

```diff
@@ -15,14 +15,37 @@ class E2fsprogs(AutotoolsPackage):
 
     license("GPL-2.0-or-later AND LGPL-2.0-or-later AND BSD-3-Clause AND MIT")
 
+    version("1.47.0", sha256="74c8ea97c73294edc6c11dc5e7fbb4324f86c28efd66ad0ba50be4eec8a48be2")
     version("1.45.6", sha256="d785164a2977cd88758cb0cac5c29add3fe491562a60040cfb193abcd0f9609b")
     version("1.45.5", sha256="0fd76e55c1196c1d97a2c01f2e84f463b8e99484541b43ff4197f5a695159fd3")
 
+    variant("fuse2fs", default=False, description="Build fuse2fs")
+
     depends_on("texinfo", type="build")
+    depends_on("fuse", when="+fuse2fs")
+
+    # fuse3 support is in the yet unreleased 1.47.1
+    patch(
+        "https://github.com/tytso/e2fsprogs/commit/5598a96.patch?full_index=1",
+        sha256="66a4aa61c8178f8d5d80563071df7f9bac234781251362d6a4613eea1d9941d0",
```

Comment:
I'm getting 72b28eb4599dbae45a01a1518ab6b8b6fc23db4f67381b49f63d3a3d45822340. Without full index I'm getting your 66a4aa61c8178f8d5d80563071df7f9bac234781251362d6a4613eea1d9941d0.
```suggestion
        sha256="72b28eb4599dbae45a01a1518ab6b8b6fc23db4f67381b49f63d3a3d45822340",
```

Reply:
concur, probably had the patch cached when I built after changing it.